talk/C++Course.tex

```diff
@@ -222,4 +222,47 @@ \section*{Index}
   \listofgodbolt
 \end{frame}
 
+\section*{External Resources and Further Reading}
+
+\begin{frame}
+  \frametitle{Books}
+  \begin{thebibliography}{10}
+    \scriptsize
+    \beamertemplatebookbibitems
+
+    \bibitem{} A Tour of C++, Third Edition
+    \newblock  Bjarne Stroustrup, Addison-Wesley, Sep 2022
+    \newblock  ISBN-13: \href{https://learning.oreilly.com/library/view/a-tour-of/9780136823575/}{978-0136816485}
+
+    \bibitem{} Effective Modern C++
+    \newblock  Scott Meyers, O'Reilly Media, Nov 2014
+    \newblock  ISBN-13: \href{https://learning.oreilly.com/library/view/effective-modern-c/9781491908419/}{978-1-491-90399-5}
+
+    \bibitem{} Clean Architecture
+    \newblock  Robert C. Martin, Pearson, Sep 2017
+    \newblock  ISBN-13: \href{https://learning.oreilly.com/library/view/clean-architecture-a/9780134494272/}{978-0-13-449416-6}
+
+    \bibitem{} The Art of UNIX Programming
+    \newblock  Eric S. Raymond, Addison-Wesley, Sep 2002
+    \newblock  ISBN-13: \href{https://learning.oreilly.com/library/view/the-art-of/0131429019/}{978-0131429017}
+
+    \bibitem{} Introduction to Algorithms, 4th Edition
+    \newblock  T. H. Cormen, C. E. Leiserson, R. L. Rivest, C. Stein, Apr 2022
+    \newblock  ISBN-13: \href{https://catalogue.library.cern/literature/ktyga-1g862}{978-0262046305}
+  \end{thebibliography}
```

Comment:
I was hesitant to add books I didn't read or didn't know much about. The exception was the book by Bjarne, because I think a book by him should be there. I will add the book on templates, and maybe the one by Jason Turner, I think his material will probably be good. I'm less inclined to add the other stuff, as I don't know (Nikolai books), and because I have actually heard bad stuff about abseil before. If someone wants to add those later, though, go for it. Thanks for the recommendations!

Reply:
The Google book has nothing to do with abseil btw, it is just mentioned there.

Reply:
I've added both the template book and Jason Turner's book to the slide. We'd need more slides to fit more stuff, so I've left the other ones out.